src/Writers/HTMLWriter.jl

```diff
@@ -1286,13 +1304,26 @@ function expand_versions(dir, versions)
 end
 
 # write version file
-function generate_version_file(versionfile::AbstractString, entries)
+function generate_version_file(versionfile::AbstractString, entries, symlinks = [])
     open(versionfile, "w") do buf
         println(buf, "var DOC_VERSIONS = [")
         for folder in entries
             println(buf, "  \"", folder, "\",")
         end
         println(buf, "];")
+
+        # The first element in entries corresponds to the latest version, but is usually not the full version
+        # number. So this essentially follows the symlinks that will be generated to figure out the full
+        # version number (stored in DOCUMENTER_CURRENT_VERSION in siteinfo.js).
+        # Every symlink points to a directory, so this doesn't need to be recursive.
+        newest = first(entries)
```

Comment:
Is this really what should be used for the link? Shouldn't it be e.g. the top entry of the user specified list or something?

Reply:
I don't see how that would work, given that the user provided `versions` argument doesn't necessarily contain any actual version numbers at all.

It might make sense to just use 
https://github.com/JuliaDocs/Documenter.jl/blob/325087c852c120227ab3d41dea6fbc7a3ac1332c/src/Writers/HTMLWriter.jl#L1236

Reply:
> doesn't necessarily contain any actual version numbers at all.

Why does that matter? I would guess most packages would like the link to direct to e.g. `stable` instead of the actual version number

Reply:
`DOCUMENTER_NEWEST` needs to be an actual version so we can compare it against `DOCUMENTER_CURRENT_VERSION` though.

Reply:
Perhaps two variables then?

Reply:
Hm, I still don't quite see what problem that solves. Why is a direct link to the full version (e.g. "v1.1.1") worse than linking to `stable` or `v1.1`, which then redirects to the full version anyways?

Reply:
Because if everyone used canonical URLs (e.g. `stable` or `v1`) things would be easier. E.g. if you copy the URL and paste somewhere

Reply:
Fair enough. Using the first entry in `entries` now, which is `stable` for the defaults.